package.json

```diff
@@ -15,14 +15,12 @@
     "format:staged": "pretty-quick --staged",
     "predev": "npm run build",
     "dev": "http-server -o example/index.html -s -c-1 -p 8080",
+    "build:ts": "tsup",
     "build:css": "cleancss src/lettercrap.css --output dist/lettercrap.min.css",
```

Comment:
Tbh I would personally go as far removing this script. I'm not sure whether minifying one css class is worth the effort..

Reply:
I am indifferent about it. Maybe we can find a way to around using CSS in general? I vote we leave it as is for now and decide on another PR focused on CSS.